tests/run-make/rustdoc-map-file/rmake.rs

```diff
@@ -8,6 +10,48 @@ fn main() {
         .arg("--generate-redirect-map")
         .out_dir(&out_dir)
         .run();
-    // FIXME (GuillaumeGomez): Port the python script to Rust as well.
-    python_command().arg("validate_json.py").arg(&out_dir).run();
+
+    let mut found_file = false;
+    read_dir_entries_recursive(&out_dir, |path| {
+        if !found_file
+            && path.is_file()
+            && path.file_name().map(|name| name == "redirect-map.json").unwrap_or(false)
+        {
+            found_file = true;
+            let generated = read_to_string(path);
+            let expected = read_to_string("expected.json");
```

Comment:
Question (following the recursive directory traversal): But AFAICT, there is only one reference file, so I'm a bit confused.

Reply:
We iterate through folders only to find the generated file. The expected one doesn't move.